Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -14889,16 +14873,16 @@
     ]
   },
   {
-    "Input": "Saldré de las 11:30 a.m. a las 12:30 a.m. 27 de diciembre",
+    "Input": "Saldré de las 11:30 a.m. a las 12:30 27 de diciembre",
```

Comment:
Does a similar sentence with am/pm that makes sense on both sider work correctly?
Can you add such spec if it doesn't currently exist?

Reply:
yes, the original sentence works correctly too, I removed the second 'a.m.' to conform the input to the corresponding English case. Added new test case.